CONTRIBUTING.md

```diff
@@ -34,6 +28,8 @@ You can install a particular branch directly from GitHub with:
 pip install git+https://github.com/weaviate/weaviate-python-client.git@BRANCH_NAME
 ```
 
+If any static analysis tools such as Pylance fail, try installing the package with:
+`--config-settings editable_mode=compat` suffix. (e.g. `pip install -e . --config-settings editable_mode=compat`)
 
 ### Testing
 
```

Comment:
(for below)

I think this is not relevant anymore, this was mainly for v3 tests. Those are all gone now